### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by kazimovzaman2** [community/community#39552] [GitHub Education] [answered]
I once signed up from the GitHub Student Developer Pack, but it got stuck in the download window, so I clicked cancel. Then when I try to register again, the sign up button does not appear.

![image](https://user-images.githubusercontent.com/92586852/202880399-f58578a4-96c7-40fd-9507-ad5b3a1b51b0.png)

Can you help me?

**Comment by DSDmark** [comment, marked as answer]
Hi @kazimovzaman2,

I think you are already **login**. That why it's not showing your.

![2022-11-20_12-47](https://user-images.githubusercontent.com/85744816/202886156-91c39c0d-f90a-4c3b-bf27-47e56a7aea8c.png)

**Reply by kazimovzaman2** [reply]
Is that mean I have already developer pack?

**Reply by DSDmark** [reply]
Yes, if it's show like above image then you have that **pack**

**Reply by kazimovzaman2** [reply]
Thank you for helping :)

**Reply by DSDmark** [reply]
Hey @kazimovzaman2,
😊 I hope you get your answer. If you get your answer, then you should close that question by click ✅ accept answer button, or you can ✍️ write the answer and accept own answer.

**Reply by Dhivakarkumar8** [reply]
DsDmark can u solve my problem please 
![Uploading IMG-20221209-WA0074.jpg…]()

**Reply by DSDmark** [reply]
Hi @Dhivakarkumar8,

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **errors** which you were facing 👍.

**Reply by tharun9522** [reply]
Hi , I have the student developer pack but how do I get access to GitHub pro ?

**Comment by Baibhav2** [comment]
It’s showing this screen when I try to signup for student pack
![Uploading 42639ECB-7F67-4B90-BB3A-77481FBFE00E.png…]()

**Reply by DSDmark** [reply]
Hi @kazimovzaman2,

> it's not good things to ask question here. this discussion already answered, so it's not caught attend, next time, if you want question, then. create new discussion.

why the way, your image link is not uploaded correctly. So, it's not showing.

**Comment by Dhivakarkumar8** [comment]
i am facing this issue of unable to confirm university and I am unable to
claim azure free trail . Its shows unable to to confirm your university I'd
.i can't do further steps without azure free trail for students

**Reply by DSDmark** [reply]
Hi @Dhivakarkumar8,

You can try these steps:-

- Check the eligibility requirements for the Developer Student Pack. On the GitHub Education website.
- Make sure you're using the correct email address.
- Contact your school's or college IT department. If you're still having trouble confirming your university.

Here is the link, all info you can get here:- [**education GitHub**](https://education.github.com/experts)

**Comment by Dhivakarkumar8** [comment]
I checked mail is correct & all details are correct. I followed all the
steps are corrected .but it's shows unable to confirm your university I'd
or your are not eligible for this pack. But my GitHub student pack is
approved .
Try to solve this issue

**Reply by DSDmark** [reply]
Wait a second, you already have GitHub student developer pack.

**Reply by DSDmark** [reply]
@Dhivakarkumar8

**Reply by DSDmark** [reply]
ok, maybe you need to read this.

> [Azure](https://www.sevenmentor.com/azure-training-institute-in-pune.php) for Students is available only to students who meet the following requirements. You must affirm that you are age 18 or older and attend an accredited, degree-granting two-year or four-year educational institution where you’re a full-time student in a STEM-related field. You must verify your academic status through your organisation’s email address . This offer is not available for use in a massive open online course (MOOC) or in other professional trainings from for-profit organisations. The Azure for Students offer is limited to one per eligible student, is non-transferable and cannot be combined with any other offers unless otherwise permitted by Microsoft. Azure for Students has a limited supply of activations and the offer is available only while supplies last. Each region has a limited number of Azure for Students offers available and your geographical location may run out of supplies while the offer is still available in other regions. Microsoft may terminate this offer at our sole discretion. Other restrictions may apply.

The last things are that. If you are not satisfied: https://azure.microsoft.com/en-us/support/

**Comment by Dhivakarkumar8** [comment]
But why I am unable to claim the free azure student pack.

**Reply by DSDmark** [reply]
@Dhivakarkumar8, Not sure but, maybe you are not fulfilled these above requirements

**Comment by Dhivakarkumar8** [comment]
No bro i fullfilled all required things .
How to claim the student pack

**Reply by stfuvaibhav** [reply]
hii bro... can you please tell me if u were able to get the access because I am having exact same issue... my application was approved but I could not sign in using my student mail... it said it is not registered with us....
